commands/utils/packagehandlers/common.go

```diff
@@ -0,0 +1,63 @@
+package packagehandlers
+
+import (
+	"fmt"
+	"github.com/jfrog/frogbot/commands/utils"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/coreutils"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+	"os/exec"
+	"strings"
+)
+
+// PackageHandler interface to hold operations on packages
+type PackageHandler interface {
+	UpdateDependency(details *utils.FixDetails) (bool, error)
```

Comment:
It looks like this function can be made private.

Reply:
That's actually the one function that should be public
update direct & indirect are private but the abstract "update" is public.